lesson5

Relaxiy opened the pull request.

Relaxiy committed:
lesson5

Relaxiy committed:
package lesson3;

import java.util.Scanner;

public class Task1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        a[0] = 1;
        a[1] = 1;
        System.out.println(a[0]);
        System.out.println(a[1]);

        for(int i = 2; i < a.length; i++) {
            a[i] = a[i - 1] + a[i - 2];
            System.out.println(a[i]);
        }

    }
}

Relaxiy committed:
package lesson3;

import java.util.Scanner;

public class Task1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        a[0] = 1;
        a[1] = 1;
        System.out.println(a[0]);
        System.out.println(a[1]);

        for(int i = 2; i < a.length; i++) {
            a[i] = a[i - 1] + a[i - 2];
            System.out.println(a[i]);
        }

    }
}

Relaxiy committed:
package lesson3;

import java.util.Scanner;

public class Task1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        a[0] = 1;
        a[1] = 1;
        System.out.println(a[0]);
        System.out.println(a[1]);

        for(int i = 2; i < a.length; i++) {
            a[i] = a[i - 1] + a[i - 2];
            System.out.println(a[i]);
        }

    }
}

damalikov commented on src/lesson5/Task1/Money.java:
```diff
@@ -0,0 +1,22 @@
+    }
+
+    public void result() {
+        EmployeeType[] employeeTypes = new EmployeeType[]{EmployeeType.DIRECTOR, EmployeeType.ENGINEER, EmployeeType.DEPARTMENT_HEAD, EmployeeType.TRAINEE};
```
У enum есть метод values,который вернет массив всех значений

damalikov commented on src/lesson5/Task2/User.java:
```diff
@@ -0,0 +1,45 @@
+    }
+
+    public String getRole() {
+        return this.role.getRole();
```
Необхоимо возвращать значение Role а не String. И по программе все сравнения выполнять именно значений enum, а не значений их внутренних полей. 
А вот в toString как раз таки необходимо вывести значение внутреннего поля.

Relaxiy committed:
package lesson3;

import java.util.Scanner;

public class Task1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        a[0] = 1;
        a[1] = 1;
        System.out.println(a[0]);
        System.out.println(a[1]);

        for(int i = 2; i < a.length; i++) {
            a[i] = a[i - 1] + a[i - 2];
            System.out.println(a[i]);
        }

    }
}

Relaxiy committed:
package lesson3;

import java.util.Scanner;

public class Task1 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        a[0] = 1;
        a[1] = 1;
        System.out.println(a[0]);
        System.out.println(a[1]);

        for(int i = 2; i < a.length; i++) {
            a[i] = a[i - 1] + a[i - 2];
            System.out.println(a[i]);
        }

    }
}
